crypto/representationproofs/representation.go

```diff
@@ -0,0 +1,148 @@
+package representationproofs
+
+import (
+	"github.com/xlab-si/emmy/crypto/common"
+	"github.com/xlab-si/emmy/crypto/groups"
+	"math/big"
+	"fmt"
+)
+
+func ProveKnowledgeOfRepresentation() bool {
+	group, err := groups.NewSchnorrGroup(256)
+	if err != nil {
+		fmt.Println(err)
```

Comment:
Please join L13 and L14 into a single `fmt.Printf()` call that prints both `err` and the message in the second call.